parser/src/Parse/Type.hs

```diff
@@ -5,82 +5,82 @@ import Text.Parsec ((<|>), (<?>), char, many1, string, try, optionMaybe)
 
 import Parse.Helpers
 import qualified Reporting.Annotation as A
-import AST.V0_16
+import qualified AST.V0_16 as AST
```

Comment:
Did qualifying this help fix warnings, or was this just an additional cleanup?

Reply:
It's for these 4 warnings:

```
[53 of 61] Compiling Parse.Type       ( parser/src/Parse/Type.hs, .stack-work/dist/x86_64-linux-nopie/Cabal-1.24.2.0/build/Parse/Type.o )

.../elm-format/parser/src/Parse/Type.hs:41:13: warning: [-Wname-shadowing]
    This binding for ‘base’ shadows the existing binding
      imported from ‘AST.V0_16’ at parser/src/Parse/Type.hs:8:1-16

.../elm-format/parser/src/Parse/Type.hs:42:14: warning: [-Wname-shadowing]
    This binding for ‘fields’ shadows the existing binding
      imported from ‘AST.V0_16’ at parser/src/Parse/Type.hs:8:1-16

.../elm-format/parser/src/Parse/Type.hs:91:24: warning: [-Wname-shadowing]
    This binding for ‘first’ shadows the existing binding
      imported from ‘AST.V0_16’ at parser/src/Parse/Type.hs:8:1-16

.../elm-format/parser/src/Parse/Type.hs:91:31: warning: [-Wname-shadowing]
    This binding for ‘rest’ shadows the existing binding
      imported from ‘AST.V0_16’ at parser/src/Parse/Type.hs:8:1-16
```

Since thse names of the exported methods from `AST.V0_16` are so generic I choose to solve it with qualifying the import. Same reasoning for the other file in this PR where` AST.V0_16` is imported.